cmd/release/cmd/tag.go

```diff
@@ -344,6 +363,19 @@ var cliTagSubCmd = &cobra.Command{
 	},
 }
 
+func copyDashboardVersions() []string {
+	versions := make([]string, len(rootConfig.Dashboard.Versions))
+
+	var i int
+
+	for version := range rootConfig.Dashboard.Versions {
```

Comment:
Can you capture the i var here from range?

Reply:
No, because `Dashboard.Versions` is a map, and range returns `key, value`